day_two_guide/topics/deprecating_node.adoc

```diff
@@ -0,0 +1,388 @@
+////
+Deprecating a node host
+
+Module included in the following assemblies:
+
+* day_two_guide/host_level_tasks.adoc
+////
+
+The procedure is the same whether deprecating an infrastructure node or an
+application node.
+
+[discrete]
+== Prerequisites
+
+Ensure enough capacity is available to migrate the existing pods from the
+node set to be removed. Removing an infrastructure node is advised only when at
+least two more nodes will stay online after the infrastructure node is removed.
+
+[discrete]
+== Procedure
+
+. List all available nodes to find the node to deprecate:
++
+----
+$ oc get nodes
```

Comment:
What is the target Openshift version  of this Document.  The output are from v3.6.

Reply:
This was written before 3.7 was released.